sympy/vector/coordsys.py

```diff
@@ -44,6 +54,7 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             Used for simple str printing.
 
         """
+        from sympy.abc import a, x, y, z
```

Comment:
It these names are used in the code calling this code, confusion may result. Dummies are unique symbols that are different from any other symbol, even those with the same name.